robotbase/src/main/java/frc/robot/Constants.java

```diff
@@ -42,4 +56,66 @@ public static final class CHOREO {
       Robot.swerve
     );
   }
+
+  public static final class ELEVATOR {
+
+    public static final SparkBaseConfig MOTOR_CONFIG_LEFT = new SparkMaxConfig()
+      .idleMode(IdleMode.kBrake)
+      .inverted(false);
+
+    public static final SparkBaseConfig MOTOR_CONFIG_RIGHT =
+      new SparkMaxConfig()
+        .idleMode(IdleMode.kBrake)
+        .follow(CAN_ID.ELEVATOR_LEFT_MOTOR, true);
+
+    public static final double LEFT_MOTOR_CLOSED_LOOP_P = 0;
```

Comment:
We can shorten these constants to just `LEFT_MOTOR_P` , the fact it is a pidf variable is indicative enough that it is for closed loop control.